src/main/webapp/app/entities/lecture/lecture.component.html

```diff
@@ -56,14 +56,13 @@ <h2 id="page-heading">
                                     <span class="d-none d-md-inline" jhiTranslate="entity.action.edit">Edit</span>
                                 </button>
                                 <button
-                                    type="submit"
-                                    [routerLink]="['/', { outlets: { popup: 'lecture/' + lecture.id + '/delete' } }]"
-                                    replaceUrl="true"
-                                    queryParamsHandling="merge"
-                                    class="btn btn-danger btn-sm"
+                                    jhiDeleteButton
```

Comment:
`*jhiHasAnyAuthority="['ROLE_ADMIN', 'ROLE_INSTRUCTOR']"` did not work here. Please double check how we can make sure that only admins and instructors of the course can see this button. Maybe we can use a similar approach as in the exercise model and set a property isAtLeastInstructor on the lecture

Reply:
`*jhiHasAnyAuthority="['ROLE_ADMIN', 'ROLE_INSTRUCTOR']"` worked for me both locally and on the test server, but I decided to use `isAtLeastInstructor` since it additionally checks if the user is in particular instructor group specified for this course. Also I noticed that the `lectureRoutes` are messed up since they allowed every user to navigate there, I changed that to only instructors and admins. As far as I understand tutors should not have access to course lectures page, right?